README.md

```diff
@@ -76,6 +76,14 @@ Then finally add the Maven shade plugin:
         </plugins>
 ```
 
+To use EffectLib via gradle just add
+
+```
+compile 'de.slikey:EffectLib:version'
```

Comment:
`implementation` is the preferred configuration to add it to, the `compile` one is deprecated in gradle.